_sass/template/epub.scss

```diff
@@ -17,6 +17,11 @@
 // Provide a variable for site output
 $output-format: "epub" !default;
 
+// Theme variables
+@import "theme/colors";
+@import "template/colors";
```

Comment:
@arthurattwell Don't we want theme colours to come after template colours, since we want template colours to be overwritten by theme colours?

If this is the case, this comment applies to the template scss files for the other output formats as well.

Reply:
@LouiseSteward Thanks for checking. This order is correct, because of the somewhat unexpected way that Sass handles precedence with `!default` values. The first `!default` wins, unlike 'normal' CSS values and non-`!default` values, where subsequent values override previous ones.

With this order, if you set a `!default` value in the theme colours, it will take precedence over the `!default` value in the template – as you correctly say they should. And then, you can override that by setting a non-`!default` value at book-level, when a given book needs a different value.

I always have to remind myself how this works, so I created a pen to demo: https://codepen.io/arthurattwell/pen/yLVzzBG